styles.css

```diff
@@ -33,23 +24,147 @@ img {
 
 a {
   color: #4da6e5;
+  cursor: pointer;
+  text-decoration: none;
 }
 
-#difference-section {
-  padding-left: 9.5rem;
-  padding-right: 9.5rem;
-  text-align: center;
+hr {
+  width: 25%;
+  margin-left: 0;
+  border: 1px solid rgb(66, 89, 111);
+  margin-bottom: 1.6rem;
+}
+
+button {
+  cursor: pointer;
+}
+
+div.footer-btn {
+  margin-left: 0.4rem;
+  border-radius: 0;
+  display: inline-block;
+  background-color: rgb(44, 134, 220);
+  padding: 0.1rem 0.3rem;
+  font-size: 0.68rem;
 }
 
 .difference-footer {
   padding-top: 2.5rem;
   padding-bottom: 6rem;
 }
 
+.difference-section-header {
+  padding-top: 3.5rem;
+  padding-bottom: 2.7rem;
+}
+
+.difference-section-subheaders {
+  padding: 0.7rem;
+}
+
+.bordered {
+  border-radius: 3px;
+}
+
+.footer-link-containers p {
+  padding-bottom: 0.5rem;
+  padding-top: 3rem;
+  color: white;
+}
+
+.col {
+  flex: 1;
+}
+
+.small-icons {
+  width: 1rem;
+  padding-right: 1rem;
+  cursor: pointer;
+}
+
+.footer-icons {
```

Comment:
Instead of custom margin on the left and right sides of the icons and legal, try putting two columns next to each other in a "sub footer" row. Text align each of them. Use margin/padding on the row instead of the columns.